templates/hooks/key/sdk_read_one_post_set_output.go.tpl

```diff
@@ -7,4 +7,9 @@
     if err != nil {
         return &resource{ko}, err
     }
-    ko.Spec.Tags = FromACKTags(tags)
\ No newline at end of file
+    ko.Spec.Tags = FromACKTags(tags)
+    keyRotationStatus, err := rm.getKeyRotationStatus(&resource{ko})
+	if err != nil {
+		return &resource{ko}, err
+	}
+	ko.Spec.EnableKeyRotation = keyRotationStatus.KeyRotationEnabled
```

Comment:
are we 100% sure that `keyRotationStatus` is not `nil` here?

Reply:
This is fixed now.